frontend/kesaseteli/employer/src/pages/index.tsx

```diff
@@ -29,12 +27,11 @@ const EmployerIndex: NextPage = () => {
 
   return (
     <Layout headingText="Työnantajan liittymä">
-      {isLoading && <p>Ladataan...</p>}
       {user && <p>Tervetuloa {user.name}!</p>}
       <Button onClick={onLogout} disabled={isLoading}>
-        Kirjaudu ulos
+        {isLoading ? 'Kirjaudutaan ulos...' : 'Kirjaudu ulos'}
```

Comment:
shouldnt it be IsLoadingLogout instead of isLoading?

Reply:
Most likely, I will try that it looks ok and text does not change during loading or anyhing.